README.md

```diff
@@ -16,25 +19,56 @@
+  - [`webpack-cli remove`](./packages/remove/README.md#webpack-cli-remove)
+  - [`webpack-cli generate-plugin`](./packages/generate-plugin/README.md#webpack-cli-generate-plugin)
+  - [`webpack-cli generate-loader`](./packages/generate-loader/README.md#webpack-cli-generate-loader)
+  - [`webpack-cli serve`](./packages/serve/README.md#webpack-cli-serve)
+  - [`webpack-cli update`](./packages/update/README.md#webpack-cli-update)
+* [webpack.config.js](https://webpack.js.org/concepts/configuration/)
+* [Contributing and Internal Documentation](#contributing-and-internal-documentation)
 
-## Migration from webpack v1 to v2
+## About
 
-The `migrate` feature eases the transition from [version 1](http://webpack.github.io/docs/) to [version 2](https://gist.github.com/sokra/27b24881210b56bbaff7). `migrate` also allows users to switch to the new version of webpack without having to extensively [refactor](https://webpack.js.org/guides/migrating/).
+Webpack CLI is a CLI tool for providing a flexible set of commands for developers to increase speed when setting up a custom webpack project. As of webpack v4, webpack is not expecting a configuration file but in many cases, developers want to create a more custom webpack configuration based on their use-cases and needs. Exactly all of these cases with webpack CLI we are providing a set of tools so improve the setup of custom webpack configuration. 
 
-`webpack-cli migrate <config>`
+### How to install
 
-[Read more about migrating](MIGRATE.md)
+When you have followed the [Getting Started](https://webpack.js.org/guides/getting-started/) guide of webpack then webpack CLI is already installed!
 
-## Creating new webpack projects
+Otherwise `npm install --save-dev webpack-cli` will install it. 
 
-The `init` feature allows users to get started with webpack, fast. Through scaffolding, people can create their own configuration in order to faster initialize new projects for various of use cases.
+### Commands
 
-`webpack-cli init webpack-scaffold-<package>`
+In order to simplify many common tasks when using webpack in your projects, webpack CLI provides different commands according to their use-cases.
 
-[Read more about scaffolding](SCAFFOLDING.md)
+webpack CLI is organized as a [multi-package repository](https://github.com/lerna/lerna) and every command is developed in the `packages` Folder.
+
+## Getting started
+
+When you have followed the [Getting Started](https://webpack.js.org/guides/getting-started/) guide of webpack then webpack CLI is already installed! Otherwise you would need to install webpack CLI and the packages you want to use. If we want to use the `init` functionality to create a new `webpack.config.js` configuration file:
+
+```sh
+npm i webpack-cli @webpack-cli/init
+npx webpack-cli init
+```
+
+After asking multiple questions related to your project and as a result create a webpack.config.js based on your selected answers. 
+
+## How it works
```

Comment:
Heading like `Scaffolding` or `Creating Addons` should work better @fokusferit

Reply:
Getting started is better

Reply:
Are you at line 65 ? @ev1stensberg